### DISCUSSION TITLE
After installation postsrsd we started to get more junk mails

### DISCUSSION THREAD

**Discussion by dengul23** [roehling/postsrsd#116] [Q&A]
After installation postsrsd we started to get more junk mails. I feel that depends on amavis configure but I don't catch where it exactly.
I use postfix as relay to Exchange.
In logs I see that amavis check emails but accept all emails, spam too. 
If stopped postsrsd service then spam stopped too.
How I can fix it?

**Comment by benchonaut** [comment]
if sending to large providers  ( especially MS, DKIM  helps with them)

* check your ip/CIDR/ASN against blacklist and do delisting if necessary
* check the srs-domain against blacklists and do delisting if necessary
* sign up to microsoft SNDS to get notified if your ip is blocked 
* engage SPF
( optional , but greatly improves the situation)
* engage DKIM
* engage DMARC

regards
